specification/storage/resource-manager/Microsoft.Storage/stable/2019-04-01/blob.json

```diff
@@ -1004,10 +1010,14 @@
             "items": {
               "$ref": "#/definitions/ListContainerItem"
             },
-            "description": "The list of blob containers."
+            "description": "List of blobs containers returned."
+          },
+          "nextLink": {
```

Comment:
Please mark this as readonly

Reply:
Addressed. Thanks for the comment!

Reply:
@huizlAzure did you push a new commit, I don't see any changes

Reply:
Yes just did. I squashed the commits maybe that's why.

Reply:
`nextLink` must be marked as `readonly`